src/wire_msg.rs

```diff
@@ -55,10 +55,10 @@ impl fmt::Display for WireMsg {
     }
 }
 
-/// Type of peer connecting to us.
+/// Type of Handshake.
 ///
-/// If the peer is a client then we allow a single connection between us with a bidirectional
-/// stream, otherwise we will have 2 connections with uni-directional stream in each.
+/// If the peer is a client then we allow a single connection between us, otherwise we will have 2
```

Comment:
Maybe worth adding this extra bit of info here?
```suggestion
/// If the peer is a client then we allow a single connection between us which will open 2 unidirectional streams for each side, otherwise we will have 2
/// connections with only one uni-directional stream in each.
```
or something like this